.storybook/deprecated/quickaction/quickaction.yml

```diff
@@ -1,13 +1,33 @@
 id: quickactions
-name: Quick actions
-status: Verified
+name: Quick actions (deprecated)
+status: Deprecated
 description: "Note that the `.spectrum-QuickActions-overlay` class should be placed on the container where the Quick Actions are displayed, and the `.spectrum-QuickActions--textOnly` class should be applied when the buttons have text only."
 SpectrumSiteSlug: https://spectrum.adobe.com/page/quick-actions/
 section:
   - name: Migration Guide
     description: |
-      ### Change workflow icon size to medium
-      Please replace `.spectrum-Icon--sizeS` with `.spectrum-Icon--sizeM`.
+      <div class="spectrum-InLineAlert spectrum-InLineAlert--negative>
+          <div class="spectrum-InLineAlert-header">
+              Deprecated component
+
+              <svg
+                  class="spectrum-Icon spectrum-Icon--sizeM spectrum-InLineAlert-icon"
+                  focusable="false"
+                  aria-hidden="true"
+                  role="img"
+                  xmlns="http://www.w3.org/2000/svg"
+                  height="36"
+                  viewBox="0 0 36 36"
+                  width="36"
+              >
+                  <path
+                      d="M17.127 2.579.4 32.512A1 1 0 0 0 1.272 34h33.456a1 1 0 0 0 .872-1.488L18.873 2.579a1 1 0 0 0-1.746 0ZM20 29.5a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-3a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Zm0-6a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-12a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Z"
+                  ></path>
+              </svg>
+          </div>
+          <div class="spectrum-InLineAlert-content">This component has been deprecated. Use an [action bar](actionbar.html) to allow users to perform actions on either a single or multiple items at the same time, instead.</div>
+      </div>
```

Comment:
Yaml is being picky about this identation:

![Screen Shot 2024-02-07 at 11 39 02 AM](https://github.com/adobe/spectrum-css/assets/23404383/e10eb540-9827-427d-86ff-679a78c2f9b5)

I think this update will work:

```suggestion
      <div class="spectrum-InLineAlert spectrum-InLineAlert--negative>
        <div class="spectrum-InLineAlert-header">
          Deprecated component
          <svg
            class="spectrum-Icon spectrum-Icon--sizeM spectrum-InLineAlert-icon"
            focusable="false"
            aria-hidden="true"
            role="img"
            xmlns="http://www.w3.org/2000/svg"
            height="36"
            viewBox="0 0 36 36"
            width="36"
          >
            <path
              d="M17.127 2.579.4 32.512A1 1 0 0 0 1.272 34h33.456a1 1 0 0 0 .872-1.488L18.873 2.579a1 1 0 0 0-1.746 0ZM20 29.5a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-3a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Zm0-6a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-12a.5.5 0 0 1 .5-.5h3a.5.5 0 0 1 .5.5Z"
            ></path>
          </svg>
        </div>
        <div class="spectrum-InLineAlert-content">This component has been deprecated. Use an [action bar](actionbar.html) to allow users to perform actions on either a single or multiple items at the same time, instead.</div>
      </div>
```

Reply:
I went ahead and moved this into the pug template so that we won't have to copy this into every new deprecated component